docs/students/working-through-a-lesson.md

```diff
@@ -1,5 +1,21 @@
-# Working through a Lesson
+# Working Through a Lesson
 
-This is Bible First. We’re all about lessons. It’s pretty easy - you just read it.
+### Step One: Read the Lesson
 
-Amet eligendi reprehenderit vitae eius et mollitia Quas aliquid quasi at reprehenderit officiis Minus dignissimos voluptatum maxime error aliquid. Ut ullam exercitationem a quae ipsam dolorem perspiciatis. Maiores voluptatibus excepturi.
+To read your lesson, [log in](https://biblefirst.online/en/sign_in) to your account and click the "Read" link at the top of the page.
+
+### Step Two: Take the Quiz
```

Comment:
Any reason you decided to go with level 3 headers here? I noticed that these don't display as sidebar links under the main page link. If they were level 2 headers, they would. But I don't know: maybe this is simpler? Thoughts?